### ISSUE TITLE
Remove SDL2 dependency

### ISSUE THREAD

**Issue by emielsteerneman** [RoboTeamTwente/roboteam#56]
I was thinking of removing SDL2 from the AI. It's only use is so that we can use the joysticks from the AI Interface, but we never use that anyway. Removing SDL2 (thus the joystick code) is also in line with the move towards the new interface, and removing anything GUI related from the AI. Thoughts?

**Comment by JAndringa** [comment]
yes agreed it's more of an inconvenience than a convenience at this point
